### DISCUSSION TITLE
Stop tinymce removing base URL

### DISCUSSION THREAD

**Discussion by tekhoi** [Laravel-Backpack/community-forum#1211] [Q&A (Help)]
Backpack V5 

Trying to stop tinymce from removing base URL and have added these options, but base url is still being removed, what am I missing?

```
CRUD::field('body')->type('tinymce')
            ->options([
               'convert_urls' => 'true',
                'relative_urls' => 'false',
                'remove_script_host' => 'false',
            ]);
```

**Comment by jcastroa87** [comment]
Hello @tekhoi 

Based on the information provided in the knowledge sources, I can offer some insights into your issue with TinyMCE in Backpack v5.

It appears that the problem you're experiencing is related to TinyMCE storing relative paths instead of full URLs for images. This issue has been addressed in a pull request for Backpack CRUD [#3281](https://github.com/Laravel-Backpack/CRUD/pull/3281). The pull request aimed to fix the problem where TinyMCE was saving relative server paths instead of full image paths, making the images unusable after fetching from the database.

However, it's worth noting that the options you're trying to set might not be working as expected due to a bug in how options were being rendered for the TinyMCE field. This was fixed in another pull request [#3258](https://github.com/Laravel-Backpack/CRUD/pull/3258), which addressed the issue of options not being properly set in the TinyMCE field declaration.

Given that you're using Backpack v5, these fixes should already be included. However, if you're still experiencing issues, you might want to try the following:

1. Ensure you're using the latest version of Backpack v5 by running `composer update backpack/crud`.

2. Clear your application cache and views:
   ```
   php artisan view:clear
   php artisan cache:clear
   ```

3. Double-check that you're not using an overridden version of the `tinymce.blade.php` file in your `app/resources` directory, as mentioned in [this issue comment](https://github.com/Laravel-Backpack/CRUD/issues/4406#issuecomment-1144654708).

4. Make sure you're setting the options correctly. The values should be boolean, not strings:

   ```php
   CRUD::field('body')->type('tinymce')
       ->options([
           'convert_urls' => true,
           'relative_urls' => false,
           'remove_script_host' => false,
       ]);
   ```

If you're still encountering issues after trying these steps, it might be worth opening a new issue on the Backpack GitHub repository or community forum, as there might be a specific problem with your setup that requires further investigation.

Cheers.

**Comment by jcastroa87** [comment]
Due to not activity, I will close this issue, but please feel free to re-open or create a new one if needed.

Cheers.
